src/custom_types/scroll_menu.py

```diff
@@ -97,11 +101,14 @@ def _get_contents(self, directory: str) -> List[Frame]:
         frames = [
             Frame(
                 Button(
-                    text=item,
-                    handler=functools.partial(self._display_content, item, directory),
+                    text=file_name,
+                    handler=functools.partial(
+                        self._display_content, file_name, directory
+                    ),
                 )
             )
-            for item in listdir(directory)
+            for file_name in listdir(directory)
+            if not file_name.startswith(".")
```

Comment:
If I'm reading right the user would still be able to create a file that starts with `.` as long as they enter a path starting from a directory inside `.thought_box` that doesn't itself start with `.` Should we prevent this?

Reply:
I believe this was addressed in 6833501c3cde74adeff3502ce271317870e08e70.

Reply:
If you're seeing another way around it, please elaborate. Or record a loom of you doing the thing.

Reply:
> If I'm reading right the user would still be able to create a file that starts with `.` as long as they enter a path starting from a directory inside `.thought_box` that doesn't itself start with `.` Should we prevent this?

This should be fixed now in f8b34c4756b1f341bf453543a6a694bc87b791de